Sources/Jobs/Middleware/TracingMiddleware.swift

```diff
@@ -37,6 +37,7 @@ public struct TracingJobMiddleware: JobMiddleware {
             }
             span.updateAttributes { attributes in
                 attributes["job.id"] = context.jobInstanceID
+                attributes["job.name"] = job.name
```

Comment:
The span is called the job name, so not sure we also need an attribute with the job name